qadence/backends/pyqtorch/backend.py

```diff
@@ -107,7 +107,7 @@ def run(
             else:
                 batch_size = max([len(tensor) for tensor in param_values.values()])
             state = circuit.native.init_state(batch_size=batch_size)
-        state = circuit.native(state, param_values)
+        state = circuit.native.run(state, param_values)
```

Comment:
Why this change?

Reply:
i like it because its clear that it returns a wf